README.MD

```diff
@@ -1,27 +1,47 @@
-# yanl - yet another native loader [![Maven Central](https://maven-badges.herokuapp.com/maven-central/fr.stardustenterprises/yanl/badge.svg)](https://maven-badges.herokuapp.com/maven-central/fr.stardustenterprises/yanl)
-
-Yet another Native library extractor/loader for the 
-[JVM](https://adoptium.net), written in [Kotlin](https://kotlinlang.org).
+# yanl - yet another native loader [![Maven Central][badge-mvnc]][yanl-mvnc]
+yet another native library loader and extractor for the [JVM][jvm], written in
+[Kotlin][kotlin].
 
 # why
-other libraries simply don't fit my needs and/or aren't as 
-expandable/configurable as I'd like.
+other libraries simply don't fit my needs and/or are not as
+expandable/configurable as we'd like.
```

Comment:
as "I'd" pas "we'd" pd ya le singulier juste au dessus reste consistant

Reply:
f4482aa